docker-compose.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Google Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3'
+
+services:
+  db:
+    image: mysql:5.7
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: 1
+      MYSQL_DATABASE: crmintapp
+      MYSQL_USER: crmintapp
+      MYSQL_PASSWORD: crmintapp
+    volumes:
+      - db_data:/var/lib/mysql
+
+  backends:
+    build:
+      context: .
+      dockerfile: docker/backends/Dockerfile
+    ports:
+      - "8080:8080"
+      - "8081:8081"
+    environment:
+      DATABASE_URI: mysql+mysqldb://crmintapp:crmintapp@db:3306/crmintapp
+      APPLICATION_ID: $APPLICATION_ID
+    volumes:
+      - ./backends/core:/app/backends/core
+      - ./backends/data:/app/backends/data
+      - ./backends/ibackend:/app/backends/ibackend
+      - ./backends/instance:/app/backends/instance
+      - ./backends/jbackend:/app/backends/jbackend
+      - ./backends/migrations:/app/backends/migrations
+      - ./backends/tests:/app/backends/tests
+    depends_on:
+      - db
+      - migrations
+      - seeds
+
+  migrations:
+    build:
+      context: .
+      dockerfile: docker/backends/Dockerfile
+    environment:
+      DATABASE_URI: mysql+mysqldb://crmintapp:crmintapp@db:3306/crmintapp
+    command: >
+      bash -c "export PYTHONPATH="/root/google-cloud-sdk/platform/google_appengine:lib" &&
+               export FLASK_APP=run_ibackend.py &&
+               export FLASK_DEBUG=1 &&
+               export APPLICATION_ID=crmintapp &&
+               cd /app/backends &&
+               python -m flask db upgrade"
+    depends_on:
+      - db
+
+  seeds:
+    build:
+      context: .
+      dockerfile: docker/backends/Dockerfile
+    environment:
+      DATABASE_URI: mysql+mysqldb://crmintapp:crmintapp@db:3306/crmintapp
+    command: >
+      bash -c "export PYTHONPATH="/root/google-cloud-sdk/platform/google_appengine:lib" &&
+               export FLASK_APP=run_ibackend.py &&
+               export FLASK_DEBUG=1 &&
+               export APPLICATION_ID=crmintapp &&
+               cd /app/backends &&
+               python -m flask db_seeds"
+    depends_on:
+      - db
+      - migrations
+
+  frontend:
+    build:
+      context: .
+      dockerfile: docker/frontend/Dockerfile
+    ports:
+      - "4200:4200"
```

Comment:
Please, add the following two lines here to make frontend as responsive to file changes as backend is:

```yaml
    volumes:
      - ./frontend/src:/app/frontend/src
```

Reply:
Good call! :)